pkg/v1alpha2/registry/consumer/consumer.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 VMware, Inc. All Rights Reserved.
```

Comment:
Need to add/update the year/license in other files as well.

Reply:
got it

Reply:
In all of them